pkg/build/graph/nodes/types.go

```diff
@@ -70,7 +70,7 @@ func BuildNodeName(o *buildapi.Build) osgraph.UniqueName {
 
 type BuildNode struct {
 	osgraph.Node
-	Build *buildapi.Build
+	*buildapi.Build
```

Comment:
Care to make a case for the change?  I'm generally against anonymous inclusion because it flattens the namespace of methods and makes the composition less obvious when reading code that uses it. ("oh, this is a buildapi.Build.  Wait, why is there an `ID` method?")